modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7180,7 +7180,7 @@ register-a-fido2-authenticator=Register a FIDO2 Authenticator
 registered-app-urls=Registered Remote App URLs
 registered-date=Registered Date
 registered-id-token-signing-alg=Registered ID Token Signing Algorithm
-registered-id-token-signing-alg-help=Set the ID token signing algorithm that is registered at OIDC provider site. The value must be one of the supported algorithms specified in discovery endpoint, or configuration entry of ID Token Signing Algorithms. The default, if omitted, is RS256.
+registered-id-token-signing-alg-help=Set the ID token signing algorithm registered with the OpenId Connect provider. The default is RS256.
```

Comment:
The purpose of the description is not about validating, but about reminding portal admin what's the configuration is for.

I am okay if you want to reduce the despcriton.